includes/Plugin.php

```diff
@@ -100,6 +100,20 @@ class Plugin {
 	 * @var Database_Upgrader
 	 */
 	public $database_upgrader;
+
+	/**
+	 * Activation Flag.
+	 *
+	 * @var Activation_Flag
+	 */
+	public $activation_flag;
+
+	/**
+	 * Activation Notice.
+	 *
+	 * @var Activation_Notice
+	 */
+	public $activation_notice;
```

Comment:
@spacedmonkey Do we **really** want to expose the `Activation_Notice` and `Activation_Flag` instances?

Reply:
We can move it private, but it makes it unhookable, which I don't love.

Reply:
I'm only wary of having to maintain backward compatibility for these.

To disable our custom plugin activation message, one could just do `add_filter( 'pre_option_web_stories_show_activation_notice', '__return_false' )`

Reply:
We can keep these part of the private api and make them public later, if that is cleaner to you.

Reply:
Sounds good to me 👍